addons/azureSets/azuresets.lua

```diff
@@ -234,7 +251,11 @@ windower.register_event('addon command', function(...)
 
         elseif comm == 'spellset' or comm == 'set' then
             if args[1] ~= nil then
-                set_spells(args[1])
+                if args[2] ~= nil then
+                    set_spells(args[1], args[2])
+                else
+                    set_spells(args[1])
+                end
```

Comment:
Good call. I did have some other logic here before, but it has since become simpler.